src/ImageSharp/Image/Image{TColor}.cs

```diff
@@ -63,6 +63,28 @@ public Image(Stream stream, Configuration configuration = null)
             this.Load(stream);
         }
 
+#if !NO_FILE_IO
```

Comment:
Maybe... I'd like to have a dig around some of the MS stuff to see what their convention is.